clustering/jgroups/src/main/java/org/jboss/as/clustering/jgroups/subsystem/JGroupsSubsystemXMLReader.java

```diff
@@ -391,6 +403,38 @@ private void parseProperty(XMLExtendedStreamReader reader, PathAddress parentAdd
         PropertyResourceDefinition.VALUE.parseAndSetParameter(reader.getElementText(), operation, reader);
     }
 
+    private void parseThreadPool(ThreadPoolResourceDefinition pool, XMLExtendedStreamReader reader, PathAddress parentAddress, Map<PathAddress, ModelNode> operations) throws XMLStreamException {
```

Comment:
Hi Tom, this is answered here: https://github.com/wildfly/wildfly/pull/6952#discussion_r20615642